### PR TITLE
[ONME-2731] K64F BR with mbedOS-5

### PR DESCRIPTION
**Pull request by hasnainvirk**
* RTOS integration
* Trace level is reduced by defualt.
  Can be changed later through mbed_app.json.
* Updating the .lib files for RTOS according to current mbed-os master

### CODE REVIEW

**Comment by hasnainvirk** [comment]
@SeppoTakalo , @TuomoHautamaki 

needs https://github.com/ARMmbed/nanostack-border-router/pull/20 too

File: nanostack-border-router.lib [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/nanostack-border-router/#9a0b032e025fb06675eb165df2a45a4ef5e15d64
```
Comment (SeppoTakalo) [inline nanostack-border-router.lib]: I don't think this is right.
You need to update this once the https://github.com/ARMmbed/nanostack-border-router/pull/20/ is in.

File: source/border_router.cpp [hunk 2]
```diff
@@ -55,7 +71,11 @@ static void toggle_led1()
  */
 static void trace_printer(const char *str)
 {
-	get_stdio_serial().printf("%s\r\n", str);
+#ifndef MBED_CONF_RTOS_PRESENT
+    pc.printf("%s\r\n", str);
+#else
+	pc.printf("%s\r\n", str);
+#endif
```
Comment (SeppoTakalo) [inline source/border_router.cpp]: Why this `#ifdef` if they both do the same thing?

**Comment by SeppoTakalo** [comment]
Once the required PR is in, please update the `.lib` and then this is OK.

Otherwise, looking good.

**Pull request merged**